WcaOnRails/app/models/competition.rb

```diff
@@ -3,9 +3,9 @@
 class Competition < ApplicationRecord
   self.table_name = "Competitions"
 
-  has_many :competition_events, dependent: :destroy
+  has_many :competition_events, -> { order(:event_id) }, dependent: :destroy
   has_many :events, through: :competition_events
-  has_many :registrations, dependent: :destroy
+  has_many :registrations, -> { order(:created_at) }, dependent: :destroy
```

Comment:
I'm doing the `order` here, doing it down there messes up with the `includes` for some reason, and create an annoying N+1 query :s

Reply:
Why do we need to sort by creation date anyway? By default those are ordered by id, which certainly resembles the order registrations were created. Even better as I can imagine two registrations created at the very same moment, whereas the default by-`id` order is obviously unambiguous.

Reply:
The default ordering comes from ActiveRecord (`ORDER BY id` was added with Rails 4 AFAIK), not from the database. Personally I wouldn't rely on ActiveRecord not changing that behavior in the future and explicitly add an `ORDER` clause. Whether `id` or `created_at` is the proper column to sort by, I don't know.

Reply:
@jonatanklosko you're totally right...

I've added an explicit order by id!